### PR TITLE
Proofreading Fixes

### PR DESCRIPTION
**Pull request by cmccandless**
- Fix various typos
- Replace fancy quotes/hyphens with standard

If it would be preferable to break this into multiple PRs by directory/language, I am willing to do this.

### CODE REVIEW

**Comment by cmccandless** [comment]
The dead link that fails the CI builds is to a file that didn't exist before this PR. Should I create an empty file to satisfy the link checker?

**Comment by iHiD** [comment]
> Should I create an empty file to satisfy the link checker?

Please. But pls put a heading in with a "TODO" as well, rather than it just being empty :)

File: languages/javascript/concept-exercises/promises/.docs/instructions.md [hunk 1]
```diff
@@ -87,16 +87,16 @@ Implement the function for premium users which fetch a translation, request it
 if it's not available, and only return it if it meets a certain threshold.
 
 ```javascript
-service.premium("jIyajbe’", 100)
-// => Promise<...> resolves "I don’t understand."
+service.premium("jIyajbe'", 100)
+// => Promise<...> resolves "I don't understand."
 
-service.premium("‘arlogh Qoylu’pu’?", 100)
+service.premium("‘arlogh Qoylu'pu'?", 100)
```
Comment (ErikSchierboom) [inline languages/javascript/concept-exercises/promises/.docs/instructions.md]: Should the first `‘` also be changed?

```suggestion
service.premium("'arlogh Qoylu'pu'?", 100)
```

File: languages/javascript/concept-exercises/promises/.docs/instructions.md [hunk 2]
```diff
@@ -87,16 +87,16 @@ Implement the function for premium users which fetch a translation, request it
...
 // => Promise<...> rejects QualityThresholdNotMet()
 
-service.premium("‘arlogh Qoylu’pu’?", 40)
+service.premium("‘arlogh Qoylu'pu'?", 40)
```
Comment (ErikSchierboom) [inline languages/javascript/concept-exercises/promises/.docs/instructions.md]:
```suggestion
service.premium("'arlogh Qoylu'pu'?", 40)
```

File: languages/javascript/concept-exercises/promises/.docs/instructions.md [hunk 3]
```diff
@@ -87,16 +87,16 @@ Implement the function for premium users which fetch a translation, request it
...
 // => Promise<...> resolves "What time is it?"
 ```
 
 ## N.B.
 
-The correct translation of `‘arlogh Qoylu’pu’?` is **How many times has it been heard?**.
+The correct translation of `‘arlogh Qoylu'pu'?` is **How many times has it been heard?**.
```
Comment (ErikSchierboom) [inline languages/javascript/concept-exercises/promises/.docs/instructions.md]:
```suggestion
The correct translation of `'arlogh Qoylu'pu'?` is **How many times has it been heard?**.
```

File: languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ First, add an abstract type `Pet` to your implementation, of which `Cat` and `Do
 
 - If a pet meets another pet that it doesn't recognize, it **is cautious**.
 
-There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids…. What happens then?
+There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids... What happens then?
```
Comment (SaschaMann) [inline languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md]:
```suggestion
There are many other things that pets may encounter that aren't pets: cars, humans, plants, natural disasters, asteroids…. What happens then?
```

Unless there's a strict requirement for files to be ASCII-only, I'd prefer if this wasn't changed. There will be many more files that require unicode in the Julia track anyway, so I don't see the point in this change.

Comment (cmccandless) [reply languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md]: The use of both the ellipsis unicode character *and* a trailing period (`….`)is incorrect. It should be changed to either:
- Three periods to compose an ASCII ellipsis (`...`) (current change)
- The ellipsis character sans-trailing period (`…`)

Either is fine with me.

Comment (SaschaMann) [reply languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md]: Ah, I didn't know that. Thanks for the correction. Let's go with the 2nd option then.

Comment (SaschaMann) [reply languages/julia/concept-exercises/multiple-dispatch/.docs/instructions.md]: I've updated the `suggestion`-block above. Apparently it's not possible to have two suggestions in one comment thread

**Comment by iHiD** [comment]
Thank you so much!

**Pull request merged**